### PR TITLE
feat: setup performance cookbook

### PR DESCRIPTION
**Pull request by arturovt**
I will rebase after #2 gets merged.

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
-    - React
-    - React Native
-    - Node 
-        - [sub-node-2](development/node/sub-node-2.md)
-        - [sub-node-1](development/node/sub-node-1.md)
+  - Angular
+    - [Style guide](development/angular/style-guide.md)
+    - [sub-ng-1](development/angular/sub-ng-1.md)
+    - [Performance cookbook](development/angular/performance-cookbook.md)
+      - [Recipes](development/angular/performance-cookbook.md)
```
Comment (bopohasii) [inline docs/_sidebar.md:13]: missing block
http://prntscr.com/o6auik

File: docs/index.html [hunk 2]
```diff
@@ -1,47 +1,50 @@
...
+        name: "recipes",
+        homepage: "https://raw.githubusercontent.com/ciklum-digital/recipes/master/README.md",
+        repo: "https://github.com/ciklum-digital/recipes",
+        loadSidebar: true,
+        coverpage: true,
+        onlyCover: true,
+        auto2top: true,
+        autoHeader: false,
+        maxLevel: 4,
+        subMaxLevel: 1,
```
Comment (bopohasii) [inline docs/index.html]: subMaxLevel: 2,

File: docs/development/angular/performance-cookbook.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+  * [Managing events](development/angular/performance-cookbook/managing-events.md)
+  * [Constant bindings and structural directives](development/angular/performance-cookbook/constant-bindings-and-structural-directives.md)
+  * [RxJS](development/angular/performance-cookbook/rxjs.md)
+  * [V8](development/angular/performance-cookbook/v8.md)
+  * [Multithreading](development/angular/performance-cookbook/multithreading.md)
+  * [Memory management](development/angular/performance-cookbook/memory-management.md)
+  * [Dynamic components](development/angular/performance-cookbook/dynamic-components.md)
+  * [Angular 8+ (Ivy)](development/angular/performance-cookbook/angular8-ivy.md)
+  * [SSR](development/angular/performance-cookbook/ssr.md)
+  * [PWA](development/angular/performance-cookbook/pwa.md)
```
Comment (bopohasii) [inline docs/development/angular/performance-cookbook.md]: I believe that TOC is excessive.
1. We have one in the sidebar and the 2nd one is duplicate
2. Docsify automatically generates submenu for header 2nd level (h2).
3. It is hard to maintain TOC in file

File: docs/development/angular/performance-cookbook/angular8-ivy.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
+# Angular 8+ (Ivy)
+
+## Table of contents
+
+* [Asynchronous components](#asynchronous-components)
+
+# Asynchronous components
```
Comment (bopohasii) [inline docs/development/angular/performance-cookbook/angular8-ivy.md]: Use only one h1 header per page
It is allowed to build a more consistent sidebar menu
But don't forget to set `subMaxLevel` to `2`

**Review by bopohasii — COMMENTED** [review]
I have finished with the review
Really a lot of work💪

**Comment by arturovt** [comment]
@bopohasii 
Could you check the update please.

File: docs/development/angular/performance-cookbook/angular8-ivy.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
+# Angular 8+ (Ivy)
+
+## Table of contents
```
Comment (bopohasii) [inline docs/development/angular/performance-cookbook/angular8-ivy.md]: same issue with TOC

please check another also

**Comment by arturovt** [comment]
@bopohasii cc

**Review by bopohasii — APPROVED** [review]

**Pull request merged**